bins/packages/golang/golang.sh

```diff
@@ -0,0 +1,32 @@
+GOLANG_VERSION="1.14.2"
```

Comment:
Instead of doing this. You can just install go using an action. The same we use in test.yml workflow